ledger-core-ethereum/test/ExplorerTests.cpp

```diff
@@ -0,0 +1,57 @@
+/*
+ *
+ * ExplorerTests
+ *
+ * ledger-core-ethereum
+ *
+ * Created by Alexis Le Provost on 04/02/2020.
+ *
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2020 Ledger
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ *
+ */
+
+#include <gtest/gtest.h>
+
+#include <ethereum/EthereumNetworks.hpp>
+#include <ethereum/api/EthereumLikeNetworkParameters.hpp>
+#include <ethereum/explorers/LedgerApiEthereumLikeBlockchainExplorer.hpp>
+
+#include <integration/ExplorerFixture.hpp>
+
+class EthereumExplorers : public ExplorerFixture<LedgerApiEthereumLikeBlockchainExplorer, api::EthereumLikeNetworkParameters> {
```

Comment:
Why plural?

Reply:
@phaazon To match with naming convention using in other tests.

Reply:
Do you have an example? I don’t find what you’re talking about.

Reply:
![Screenshot 2020-03-12 at 17 12 03](https://user-images.githubusercontent.com/6042495/76542006-b302fe80-6484-11ea-84a5-5cbb64efa038.png)
![Screenshot 2020-03-12 at 17 12 32](https://user-images.githubusercontent.com/6042495/76542009-b302fe80-6484-11ea-865b-c1ef64e930e1.png)

Reply:
Thanks! That’s a weird convention but I agree with your decision. Good call.